corelib/src/libs/SireMove/openmmfrenergyst.cpp

```diff
@@ -2932,8 +3538,6 @@ void OpenMMFrEnergyST::createContext(IntegratorWorkspace &workspace, SireUnits::
             integrator_openmm = new OpenMM::VariableVerletIntegrator(integration_tol); //integration tolerance error unitless
         else if (Integrator_type == "langevin")
             integrator_openmm = new OpenMM::LangevinIntegrator(converted_Temperature, converted_friction, dt);
-        else if (Integrator_type == "langevinmiddle")
-            integrator_openmm = new OpenMM::LangevinMiddleIntegrator(converted_Temperature, converted_friction, dt);
```

Comment:
It looks like your merge might have deleted some of the recent additions. Support for the `langevinmiddle` integrator was recently added by @msuruzhon. I'll have check through to see if anything else is missing.